### PR TITLE
feat-mobile: onboarding stronghold import export

### LINKED ISSUE #5063: [Task]: Onboarding: Update routers for stronghold import
### Task description

TBD

### Requirements

TBD

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #5064: [Task]: Onboarding: Add stronghold import view
### Task description

TBD

### Requirements

 TBD

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #5065: [Task Tracker]: Stronghold import
### Task description

- [ ] #5063
- [ ] #5064

### Requirements

 TBD

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #5577: [Task]: Add plugin to import file from system (Android)
### Task description

At the moment is not possible to import a file from system. We should add the necessary plugins to enable this feature (needed to import profile from stronghold).

Depends on https://github.com/iotaledger/firefly/pull/5498

### Requirements

Add plugin to import file from system, Android

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #5578: [Task]: Add plugin to import file from system (iOS)
### Task description

At the moment is not possible to import a file from system. We should add the necessary plugins to enable this feature (needed to import profile from stronghold).

Depends on https://github.com/iotaledger/firefly/pull/5498

### Requirements

Add plugin to import file from system, iOS

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #5579: [Task]: Add plugin to export file by sharing (Android)
### Task description

At the moment is not possible to export/share a file from system. We should add the necessary plugins to enable this feature (needed to export stronghold, tx history,...).

Depends on https://github.com/iotaledger/firefly/pull/6245

### Requirements

Add plugin to share (export) file, Android

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #5580: [Task]: Add plugin to export file by sharing (iOS)
### Task description

At the moment is not possible to export/share a file from system. We should add the necessary plugins to enable this feature (needed to export stronghold, tx history,...).

Depends on https://github.com/iotaledger/firefly/pull/6245

### Requirements

Add plugin to share (export) file, iOS

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by amadeu2**
## Summary

This PR adds the Capacitor filesystem access plugin allowing to import / export the Stronghold file at onboarding views

## Changelog

```
- Please write a list of changes
```

## Relevant Issues
Closes #5063 
Closes #5064 
Closes #5065 
Closes #5577 
Closes #5578 
Closes #5579 
Closes #5580

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows
-   **Mobile**
    -   [ ] iOS
    -   [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/mobile/features/features.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ const features = {
         security: {
             enabled: true,
             exportStronghold: {
-                enabled: false,
+                enabled: true,
```
Comment (begonaalvarezd) [inline packages/mobile/features/features.ts:91]: this should be kept to `false` until we add the frontend code 🙏🏼

File: packages/mobile/capacitor/capacitorApi.ts [hunk 2]
```diff
@@ -417,17 +424,17 @@ export const CapacitorApi = {
...
-        // }
-        // if (os === 'ios') {
-        //     void SecureFilesystemAccess.revokeAccess()
-        // }
+        const os: string = Capacitor.getPlatform()
+        const { selected } = await SecureFilesystemAccess.showPicker({
+            type: 'folder',
+            defaultPath: '',
+        })
+        if (os === 'ios') {
```
Comment (paul-boegelsack) [inline packages/mobile/capacitor/capacitorApi.ts:432]: I guess we do not need both if statements, right? Do we even need the function "saveRecoveryKit" for mobile?

File: packages/mobile/capacitor/capacitorApi.ts [hunk 3]
```diff
@@ -130,6 +133,10 @@ export const CapacitorApi = {
     //     }
     // },
 
+    copyFile(sourceFilePath, destinationFilePath) {
```
Comment (paul-boegelsack) [inline packages/mobile/capacitor/capacitorApi.ts:136]: Is this a placeholder?

File: packages/mobile/capacitor/capacitorApi.ts [hunk 4]
```diff
@@ -27,25 +27,28 @@ import { PincodeManager } from '../../mobile/capacitor/lib/pincodeManager'
...
 // const WalletApi = IotaWalletMobile
-// import { WalletApi } from '@iota/wallet-mobile'
-
-import * as WalletApi from './wallet-api'
+import { Account, AccountManager } from '@iota/wallet-mobile'
+const WalletApi = {
+    Account,
+    AccountManager
+}
+// import * as WalletApi from './wallet-api'
```
Comment (paul-boegelsack) [inline packages/mobile/capacitor/capacitorApi.ts:35]: Do we still need the comments?

**Pull request closed without merging**